utils/linux-install-luarocks.sh

```diff
@@ -38,10 +40,11 @@ fi
 ./configure $WITH_LUA_OPT \
     > build.log 2>&1 || (cat build.log && exit 1)
 
+rm -rf luarocks-"$LUAROCKS_VER"
```

Comment:
delete before `make build` and `make install`?

Reply:
fixed